doc/api/dns.md

```diff
@@ -665,6 +666,19 @@ That is, if attempting to resolve with the first server provided results in a
 subsequent servers provided. Fallback DNS servers will only be used if the
 earlier ones time out or result in some other error.
 
+## `dns.verbatim`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Determines the default value of `verbatim` when it's not passed to
+[`dns.lookup()`][] or [`dnsPromises.lookup()`][]. Defaults to `false`.
+
+Configurable using the [`--dns-order`][] CLI option. Change `dns.verbatim`
+will override the [`--dns-order`][] option.
+
```

Comment:
Is a change in the value here propagated to worker threads?

Reply:
Yes, as both `child_process` and `worker_threads` inherit `execArgv` from parent process by default. This is expected 
behavior (for my own scene at least).

Reply:
Doesn't it just set a local `defaultVerbatim` copy below?

Reply:
Sorry, I misunderstood the question. Yes, `dns.verbatim = true` won't propagate.

Reply:
I have deleted the description in avoid of been misunderstood.

Reply:
Ok, so there's a definite discussion to be had then about whether it *should* be inherited. I don't think it's a blocking issue for this PR right now but I would be curious what @mcollina and @addaleax think about it.

Reply:
I also think the CLI option should take precedence over dns.verbatim.
I.e. `{ verbatim: }` > `--dns-order` > `dns.verbatim`.